content/teaching-curriculum/chapter1_high_standards

```diff
@@ -0,0 +1,32 @@
+---
+_db_id: 
```

Comment:
please leave out _db_id. It gets filled in automatically. Rather dont even put it there.

Reply:
Done. W.r.t. overall comments: 
- frontmatter: I wasn't sure which fields are required. I removed anything that was null / similar. 
- The > symbols were there from my own planning. I managed to push an older version somehow. Should now be in full sentence format. 
- This was never meant to be a comprehensive curriculum. It's something very quick to get the team started on getting familiar with the techniques in the book. The real value will come when we start implementing various techniques into the everyday system. || I was hoping to just get it up quickly to get them started, but the hugo/syllabus-repo setup has slowed things down.